### DISCUSSION TITLE
Got error with 0.52.0-beta5.9

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#524] [General]
Hi,

When starting this beta I got an error, but it seems the server will start anyway. See my screenshot below PSE.

Kind regards,
Henk -  https://fileserver.ph5hp.nl
![image](https://github.com/rejetto/hfs/assets/140949730/1d2b282f-c34a-4932-ba09-64dbec4f8931)

**Comment by rejetto** [comment]
what Windows version do you have?

**Reply by PH5HP** [reply]
Hi,

Windows 10 64bit ...

**Comment by rejetto** [comment]
hfs is running the command that you see starting with `@chcp`, the whole line, but it gets an error.
That command is used to determine if we are running as a service or not.
This is a recent addition. It's not essential, but hfs can use this info to behave accordingly.
It is possible that there's something wrong with your system, but even in case, I need to work around this problem somehow.

**Reply by PH5HP** [reply]
Hi,

That is possible, but I have no other problems with my Windows 10 64bit and this Windows is has been installed just 8 months ago... I have no idea what the meaning of this error is.

Kind regards,
Henk - https://fileserver.ph5hp.nl

**Comment by rejetto** [comment]
can you try running yourself in the terminal `@chcp 65001 ›nul & cmd /c wmic service get PathName,DisplayName, ProcessId /value` and report here the result?

**Reply by PH5HP** [reply]
Hi,

Terminal? ... Is that the window where I made the screenshot? I tried to run the command you wanted me to run, but that did not work.

Kind regards,
Henk - https://fileserver.ph5hp.nl/
![image](https://github.com/rejetto/hfs/assets/140949730/cceb3466-292f-4cc1-b23d-f088c7da4d18)

**Comment by rejetto** [comment]
sorry, with terminal i meant the command prompt, also known as shell, or cmd

**Comment by PH5HP** [comment]
![image](https://github.com/rejetto/hfs/assets/140949730/0c188ef6-fa40-4a26-ae95-68b974dc0923)

**Comment by rejetto** [comment]
i'm sorry,
there's a typo in the character >
this should be correct

`@chcp 65001 > nul & cmd /c wmic service get PathName,DisplayName, ProcessId /value`

**Reply by PH5HP** [reply]
![image](https://github.com/rejetto/hfs/assets/140949730/edd40fc3-f040-4d6f-b8e5-95996ec32dff)

**Comment by rejetto** [comment]
can please try this version?
`wmic service get PathName,DisplayName,ProcessId /value`

**Reply by PH5HP** [reply]
Same output: Invalid XML content.

![image](https://github.com/rejetto/hfs/assets/140949730/fd24dd4a-c99a-4512-922f-ef4eb4299ded)

**Comment by rejetto** [comment]
does this solve it?
`wmic service get PathName,ProcessId /value`

**Reply by PH5HP** [reply]
No, sorry ... Same output...

**Comment by rejetto** [comment]
:(
please one more
`wmic service get ProcessId`

**Reply by PH5HP** [reply]
A lot of output now. See below. To big for a screenshot .. So just raw data:
----------------------------------------------------------------------------------

C:\Users\Henk>wmic service get ProcessId
ProcessId
2228
0
0
6204
7156
0
6888
0
0
0
10104
0
0
3184
5200
0
5624
6508
0
13288
0
0
3996
0
0
5640
0
1284
12984
0
11408
0
16000
7164
7740
0
0
0
0
2148
5456
0
1284
0
0
3240
0
0
1600
0
0
6240
0
21856
2624
0
0
0
2844
5616
0
7140
1648
0
5388
0
18020
0
7208
0
0
0
0
0
1828
3120
0
0
3976
0
3548
0
0
26356
14180
0
0
0
0
0
0
24528
0
0
1368
1696
0
0
0
0
5300
0
7756
1176
0
6288
6860
17796
7796
0
0
1728
1424
0
0
0
0
0
7300
0
5640
0
0
0
0
6456
0
0
1904
3032
0
0
2484
0
0
0
0
2340
2020
6640
3844
0
6452
7276
0
0
16572
0
0
0
0
0
1284
6488
7408
0
0
8800
1284
0
1896
0
0
0
7416
0
0
0
3684
1372
0
1372
7420
1176
0
0
1856
0
8096
0
0
15240
2356
3984
0
0
0
0
7216
0
4652
0
0
5448
0
0
0
0
0
0
6700
0
3968
0
7260
4792
18712
0
6132
0
0
3952
1284
5008
0
7668
0
3960
0
1912
4464
7240
0
0
0
0
15052
0
0
1140
10628
0
1176
0
0
7284
0
0
0
0
0
0
0
0
7336
7428
7328
0
0
0
0
0
0
0
0
0
0
0
5392
11416
7748
0
0
0
0
0
0
0
0
0
0
2736
3084
0
0
0
0
0
0
0
7316
0
0
0
0
7292
12824
7436
0
0
0
0
0
0
6252
0
0
0
0
9884
4300
0
0
0
0
0
0
25624
0
0
0
0
0
26924


C:\Users\Henk>

----------------------------------------------------------------------------

**Comment by rejetto** [comment]
cool, thanks!

**Comment by rejetto** [comment]
this version is supposed to fix your error
[0.52-beta5.94](https://github.com/rejetto/hfs/files/14652750/hfs-windows.zip)

**Reply by PH5HP** [reply]
Hi,

No error...  It seems OK now :-)

Kind regards,
Henk - https://fileserver.ph5hp.nl/

**Reply by rejetto** [reply]
thank you again!
